spec/templates_compiler/converter_generator_spec.rb

```diff
@@ -144,6 +152,15 @@ def set_local_variables(binding, vars)
             inline_image: '"<inline_image>"',
           }}
 
+          it 'returns true when the converter handles a transformation' do
+            output = instance.generate.string
+            eval(output) # rubocop:disable Security/Eval
```

Comment:
@jirutka I'm using `eval` to evaluate the generated code to check that the `handles?` method on the converter is working. I'm not really sure how to test it otherwise... but if you do have an idea, please feel free to share it so I can use it instead.

Reply:
There are some sandboxing libraries for Ruby that makes eval less evil, but I think that this is okay for the test.